src/Datadog.Trace.DuckTyping/README.md

```diff
@@ -0,0 +1,757 @@
+|       GetProtectedField |    .NET 4.7.2 |        Getter |   Public | 3.424 ns | 0.0471 ns | 0.0393 ns |  1.00 |    0.00 |     - |     - |     - |         - |
+|       GetProtectedField | .NET Core 3.1 |        Getter |   Public | 3.232 ns | 0.0316 ns | 0.0264 ns |  0.94 |    0.01 |     - |     - |     - |         - |
+|                         |               |               |          |          |           |           |       |         |       |       |       |           |
+|         GetPrivateField |    .NET 4.7.2 |        Getter |   Public | 3.384 ns | 0.0300 ns | 0.0251 ns |  1.00 |    0.00 |     - |     - |     - |         - |
+|         GetPrivateField | .NET Core 3.1 |        Getter |   Public | 3.338 ns | 0.0897 ns | 0.0839 ns |  0.98 |    0.03 |     - |     - |     - |         - |
+|                         |               |               |          |          |           |           |       |         |       |       |       |           |
+|          SetPublicField |    .NET 4.7.2 |        Setter |   Public | 2.078 ns | 0.0375 ns | 0.0333 ns |  1.00 |    0.00 |     - |     - |     - |         - |
+|          SetPublicField | .NET Core 3.1 |        Setter |   Public | 2.396 ns | 0.0510 ns | 0.0477 ns |  1.15 |    0.03 |     - |     - |     - |         - |
+|                         |               |               |          |          |           |           |       |         |       |       |       |           |
+|        SetInternalField |    .NET 4.7.2 |        Setter |   Public | 3.402 ns | 0.0424 ns | 0.0354 ns |  1.00 |    0.00 |     - |     - |     - |         - |
+|        SetInternalField | .NET Core 3.1 |        Setter |   Public | 3.146 ns | 0.0248 ns | 0.0207 ns |  0.92 |    0.01 |     - |     - |     - |         - |
+|                         |               |               |          |          |           |           |       |         |       |       |       |           |
+|       SetProtectedField |    .NET 4.7.2 |        Setter |   Public | 3.368 ns | 0.0443 ns | 0.0370 ns |  1.00 |    0.00 |     - |     - |     - |         - |
+|       SetProtectedField | .NET Core 3.1 |        Setter |   Public | 3.167 ns | 0.0242 ns | 0.0189 ns |  0.94 |    0.01 |     - |     - |     - |         - |
+|                         |               |               |          |          |           |           |       |         |       |       |       |           |
+|         SetPrivateField |    .NET 4.7.2 |        Setter |   Public | 3.392 ns | 0.0563 ns | 0.0499 ns |  1.00 |    0.00 |     - |     - |     - |         - |
+|         SetPrivateField | .NET Core 3.1 |        Setter |   Public | 3.171 ns | 0.0663 ns | 0.0621 ns |  0.94 |    0.02 |     - |     - |     - |         - |
+
+### Properties Getter and Setter
+
+The `proxy` column indicates the target type access modifier.
+
+Tests with `blank` proxy are the direct access of the value through an interface without creating any ducktype proxy.
```

Comment:
Does this text also apply to the above `Fields Getter and Setter` section? If so, can you copy & paste this up there too?

Reply:
Yeah, you're right, done in https://github.com/DataDog/dd-trace-dotnet/pull/946/commits/564fcdf77e48b4645b9881962e5bcd6dd38ee9f5